splunk-quarkus/pr_check.sh

```diff
@@ -18,6 +19,15 @@ IMAGE="quay.io/cloudservices/eventing-integrations-splunk"
 # Install bonfire repo/initialize
 CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
 curl -s $CICD_URL/bootstrap.sh > .cicd_bootstrap.sh && source .cicd_bootstrap.sh
-source $CICD_ROOT/build.sh
-source $APP_ROOT/deploy_ephemeral_env.sh
+source "$CICD_ROOT/build.sh"
+source "$CICD_ROOT/deploy_ephemeral_env.sh"
 #source $CICD_ROOT/cji_smoke_test.sh
+
+# Need to make a dummy results file to make tests pass
+cd ../..
```

Comment:
this is probably not a good idea (to cd around I mean). 

I'd remove that if possible. 

Also would use an absolute path for the test artifacts location to ensure I'm putting them in the right place, such as 

```
$APP_ROOT/artifacts/junit-dummy.xml
```

probably all that's happening is that it's in the wrong place so the teardown artifact collection operation is not finding the reports

Reply:
@vkrizan  I just saw the only problem is the location of the artifact , see SUCCESS build from  #22